guide/14-deep-learning/exploring_model_extensibility_for_named_entity_recognition.ipynb

```diff
@@ -0,0 +1,722 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        "ModelType": "Transformer",
</pre><p>In the above statement you said ModelType should be _TransformerEntityRecognizer but here its mentioned as Transformer</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Fixed it</p>